.github/actions/version_checking/action.yml

```diff
@@ -0,0 +1,36 @@
+name: version_checking
+description: 'Сhecks the equality of version from code and the last repository tag'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Clone the module
+      uses: actions/checkout@v2
+
+    - name: Setup tarantool
+      uses: tarantool/setup-tarantool@v1
+      with:
+        tarantool-version: '2.9'
+
+    - name: Package version
+      # Found that Github checkout Actions pulls all the tags, but
+      # right it deannotates the testing tag, check:
+        #  https://github.com/actions/checkout/issues/290
+      # But we use 'git describe ..' calls w/o '--tags' flag and it
+      # prevents us from getting the needed tag for packages version
+      # setup. To avoid of it, let's fetch it manually, to be sure
+      # that all tags will exists always. The actions/checkout depth
+      # does not help.
+      run: |
+        git fetch --tags -f
+          tarantoolctl rocks make
```

Comment:
To be honest, I'm thinking that it would be better as a separate job rather than step. In this case, we'll need to rework queue CD to have separate "build a package" and "publish a package" workflows. It is a bit of extra work. Also it is troublesome for Enterprise CD since we cannot use GitHub artifacts due to limits for private repos. We'll need to think about this question.